src/main/java/com/exactpro/th2/conn/MicroserviceMain.java

```diff
@@ -215,30 +214,35 @@ private static Object getParamValue(ISettingsProxy settings, String parameterNam
         LOGGER.info("AvailableProcessors '{}'", Runtime.getRuntime().availableProcessors());
 
         return flowable
-                .doOnNext(message -> LOGGER.trace(
-                        "Message before observeOn with sequence {} and direction {}",
-                        message.getSequence(),
-                        message.getDirection()
-                ))
+                .doOnNext(message -> {
+                	LOGGER.trace(
+							"Message before observeOn with sequence {} and direction {}",
+							message.getSequence(),
+							message.getDirection()
+					);
+				})
                 .observeOn(PIPELINE_SCHEDULER)
                 .doOnNext(connectivityMessage -> LOGGER.debug("Start handling connectivity message {}", connectivityMessage))
-                .groupBy(ConnectivityMessage::getDirection)
+				.groupBy(ConnectivityMessage::getSessionAlias)
```

Comment:
There is not need for grouping because the is only one session alias in the whole conn component